internal/backends/linuxkms/display/egldisplay.rs

```diff
@@ -118,12 +118,27 @@ impl raw_window_handle::HasDisplayHandle for EglDisplay {
 }
 
 pub fn create_egl_display() -> Result<EglDisplay, PlatformError> {
+    let drm_devices = 
+        std::fs::read_dir("/dev/dri/")
+            .map_err(|e| format!("Error opening /dev/dri/: {e}"))?;
```

Comment:
cargo fmt suggests this:

```suggestion
        std::fs::read_dir("/dev/dri/").map_err(|e| format!("Error opening /dev/dri/: {e}"))?;
```